test-resources-server/src/main/java/io/micronaut/testresources/server/TestResourcesService.java

```diff
@@ -54,7 +55,8 @@ public static void main(String[] args) {
                 }
             });
         long dur = System.nanoTime() - sd;
-        LOGGER.info("A Micronaut Test Resources server is listening on port {}, started in {}ms", context.getBean(EmbeddedServer.class).getPort(), Duration.ofNanos(dur).toMillis());
+        LOGGER.info("A Micronaut Test Resources server is listening on port {}, started in {}ms",
```

Comment:
Can we include a link to the control panel if available?

Reply:
I'm not sure. We actually removed the test resources server link at startup because it was confusing to users (they can mix it with their own application server).